.github/workflows/label.yml

```
@@ -0,0 +1,134 @@
+        startsWith(github.event.pull_request.title, 'refactor') ||
+        startsWith(github.event.pull_request.title, 'perf') ||
+        startsWith(github.event.pull_request.title, 'test') ||
+        startsWith(github.event.pull_request.title, 'chore')
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN}}
+        labels: enhancement
+
+    - uses: actions-ecosystem/action-add-labels@v1
+      if: |
+        startsWith(github.event.pull_request.title, 'build') ||
+        startsWith(github.event.pull_request.title, 'revert') ||
+        startsWith(github.event.pull_request.title, 'maint')
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN}}
+        labels: maintenance
+      
+    - uses: actions-ecosystem/action-add-labels@v1
+      if: |
+        startsWith(github.event.pull_request.title, 'doc') || 
+        startsWith(github.event.pull_request.title, 'docs')
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN}}
+        labels: documentation
+
+    - uses: actions-ecosystem/action-add-labels@v1
+      if: |
+        startsWith(github.event.pull_request.title, 'docker') ||
+        startsWith(github.event.pull_request.title, 'no-ci') ||
+        startsWith(github.event.pull_request.title, 'ci')
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN}}
+        labels: CI/CD
+
+  commenter:
+    runs-on: ubuntu-latest
+    needs: [labeler]
+    permissions:
+      pull-requests: write
+    steps:
+    - name: Suggest to add labels
+      uses: peter-evans/create-or-update-comment@v4
+      # Execute only when no labels have been applied to the pull request
+      if: toJSON(github.event.pull_request.labels.*.name) == '{}' # Empty labels
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        issue-number: ${{ github.event.pull_request.number }}
+        body: |
+          Please add one of the following labels to add this contribution to the Release Notes :point_down:
+          - [bug](https://github.com/ansys/pyfluent/pulls?q=label%3Abug+)
+          - [new feature](https://github.com/ansys/pyfluent/pulls?q=label%3A"new%20feature"+)
+          - [enhancement](https://github.com/ansys/pyfluent/pulls?q=label%3Aenhancement+)
+          - [documentation](https://github.com/ansys/pyfluent/pulls?q=label%3Adocumentation+)
+          - [examples](https://github.com/ansys/pyfluent/pulls?q=label%3Aexamples+)
+          - [maintenance](https://github.com/ansys/pyfluent/pulls?q=label%3Amaintenance+)
+          - [CI/CD](https://github.com/ansys/pyfluent/pulls?q=label%3Aci%2Fcd+)
+
+  changelog-fragment:
+    name: "Create changelog fragment"
+    needs: [labeler]
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: ansys/actions/doc-changelog@v8
+      with:
+        bot-user: ${{ secrets.PYANSYS_CI_BOT_USERNAME }}
+        bot-email: ${{ secrets.PYANSYS_CI_BOT_EMAIL }}
+        token: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
      with:
        bot-user: ${{ secrets.PYANSYS_CI_BOT_USERNAME }}
        bot-email: ${{ secrets.PYANSYS_CI_BOT_EMAIL }}
        token: ${{ secrets.GITHUB_TOKEN }}
        use-conventional-commits: true
        use-default-towncrier-config: true
```
I would recommend also using conventional commits convention (since you are already following it) and activate the default-towncrier-config option as well so that if any update is done to our default config, you also get it.